src/components/ui/featured-campaign.js

```diff
@@ -76,8 +76,12 @@ function FeaturedCampaign({ loading, data, error }) {
         <Box as={ChevronLeft} />
       </Box>
       {data.campaigns.campaigns.map((campaign, campaignIndex) => {
-        const currentFund = parseInt(campaign?.totalFunds, 10);
-        const totalPercent = (currentFund * 100) / campaign?.campaignTarget;
+        const currentFund = campaign.totalFunds
```

Comment:
selam @erhnml, göndermiş olduğun katkı için teşekkürler.

burayı tek satır'a indirmek ve karışıklığı biraz daha aza indirmek adına `const currentFund = parseInt(campaign?.totalFunds || 0, 10)` şeklinde yazmaya ne dersin?

Reply:
Evet cok mantikli olur, tesekkurler.